src/content.js

```diff
@@ -248,14 +248,13 @@ function addDeleteForkLink() {
 	const postMergeDescription = select('#partial-pull-merging .merge-branch-description');
 
 	if (postMergeDescription) {
-		const currentBranch = postMergeDescription.querySelector('.commit-ref.current-branch');
+		const currentBranch = postMergeDescription.querySelector('.commit-ref');
 		const forkPath = currentBranch ? currentBranch.title.split(':')[0] : null;
 
 		if (forkPath && forkPath !== repoUrl) {
 			postMergeDescription.append(
 				<p id="refined-github-delete-fork-link">
```

Comment:
I think we can avoid this `p`, just use the `id` on the link and center it via CSS